download-build-run-locale-screenshots.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+HOME=""
+REPO="https://github.com/mozilla-mobile/firefox-tv.git"
+CLONE_DIRECTORY="$HOME/firefox-tv"
+SCREENSHOTS_DIR="$HOME/firefox-tv/fastlane"
+CONFIG="$HOME/Screengrabfile"
+#DROPBOX_DIR="/Users/firefoxtv/Dropbox"
+
+# export target Amazon Fire TV serial (AFTT 5.1.1)
```

Comment:
For the values where user needs to fill in their own, let's add <INPUT NEEDED> or something like that